io.github.Youda008.DoomRunner.yaml

```diff
@@ -0,0 +1,96 @@
+app-id: io.github.Youda008.DoomRunner
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: doomrunner.sh
+finish-args:
+  - --device=all
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  # Allow to execute host engines
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
this is not allowed by default.

Also why ?

Reply:
See my comments below.